drivers/flash/nrf_qspi_nor.c

```diff
@@ -536,19 +536,38 @@ static int qspi_nor_read(struct device *dev, off_t addr, void *dest,
 static int qspi_nor_write(struct device *dev, off_t addr, const void *src,
 			  size_t size)
 {
+	const void *sptr = src;
+	size_t dlen = size;
+	u8_t buf[4];
```

Comment:
There's a risk this buffer might not be aligned as required.  I'm updating the upstream PR to include a fix.

Reply:
Link?

Reply:
Added the align fix.